src/test/java/teammates/test/cases/browsertests/FeedbackRubricQuestionUiTest.java

```diff
@@ -82,14 +89,14 @@ private void testInstructorResultsPage() throws Exception {
         instructorResultsPage.waitForPanelsToExpand();
         instructorResultsPage.verifyHtmlMainContent("/instructorFeedbackResultsPageRubricGQRView.html");
         
-        // Recipient Giver Question View
+        // Recipient Question Giver View
```

Comment:
My apologies, I should have mentioned that it was best to avoid unrelated changes 😓. I thought I'd leave it as you had already made the change.